web/styles/dark_theme.css

```diff
@@ -59,11 +59,6 @@
         border: 1px solid hsl(0deg 0% 0% / 60%);
     }
 
-    .modal__btn:disabled {
-        opacity: 1;
-        background-color: hsl(0deg 0% 83% / 50%);
-    }
```

Comment:
Moved this below since we want same CSS for compose buttons in disabled state as well and here we only define CSS for modals.